### PR TITLE
Add a flag to skip binaries download

### LINKED ISSUE #17: Do not overwrite binaries from released images
Now we are overwriting the binaries with versions that the playbook can reach. In the images we ship, we need to make sure that the version we are testing matches the version in the image, for this we need to add a flag that will disable the download and install binaries step

### PR DESCRIPTION
**Pull request by Alexpast1423**
Closes: #17

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -62,3 +62,6 @@ neofs_s3__combined_config: '{{
   neofs_s3__config +
   neofs_s3__group_config +
   neofs_s3__host_config }}'
+
+# Flag to skip binaries download
+neofs_ir__binaries_download: True
```
Comment (anatoly-bogatyrev) [inline defaults/main.yml]: neofs_ir__binaries_download -> neofs_s3__binaries_download

File: tasks/binary.yml [hunk 2]
```diff
@@ -9,6 +9,7 @@
     group: 'root'
     mode: '0755'
   notify: [ 'Restart NeoFS S3 service' ]
+  when: neofs_storage__binaries_downloads | bool
```
Comment (anatoly-bogatyrev) [inline tasks/binary.yml]: neofs_storage__binaries_downloads -> neofs_s3__binaries_downloads

File: tasks/binary.yml [hunk 3]
```diff
@@ -18,3 +19,4 @@
     owner: 'root'
     group: 'root'
     mode: '0755'
+  when: neofs_storage__binaries_downloads | bool
```
Comment (anatoly-bogatyrev) [inline tasks/binary.yml]: neofs_storage__binaries_downloads ->  neofs_s3__binaries_downloads

**Review by anatoly-bogatyrev — CHANGES_REQUESTED** [review]

File: defaults/main.yml (hunk 1 above)
Comment (realloc) [inline defaults/main.yml]: Add newline here, please

**Comment by realloc** [comment]
Please add Sign-Off line in commit description.

**Comment by Alexpast1423** [comment]
Ok, re-review now, please.

**Review by realloc — APPROVED** [review]

**Review by anatoly-bogatyrev — APPROVED** [review]

**Pull request merged**